package.json

```diff
@@ -107,5 +100,6 @@
     "npm": {
       "skipChecks": true
     }
-  }
+  },
+  "packageManager": "pnpm@10.12.1+sha512.f0dda8580f0ee9481c5c79a1d927b9164f2c478e90992ad268bbb2465a736984391d6333d2c327913578b2804af33474ca554ba29c04a8b13060a717675ae3ac"
```

Comment:
Can we remove this? Our repo is currently using npm (in github scripts etc). If we want to switch to pnpm, we can do it in a separate PR.

Reply:
Yes, apologies, I didn't mean for that to go in.

Reply:
Updated